backend/account_v2/organization.py

```diff
@@ -32,13 +29,6 @@ def create_organization(
                 organization_id=organization_id,
             )
             organization.save()
-
-            for subscription_plugin in subscription_loader:
```

Comment:
@athul-rs  can you point me to the code where `add` is getting invoked post this change?